src/main/java/org/libsdl/app/SDLActivity.kt

```diff
@@ -337,8 +337,6 @@ open class SDLActivity internal constructor (context: Context?) : RelativeLayout
             return
         }
 
-        nativeDestroyScreen()
```

Comment:
Does moving the app in background and foreground still work?

Reply:
yes it does 👍